### PR TITLE
Remove WAFS from ops

### PR DESCRIPTION
**Pull request by aerorahul**
# Description
WAFS is being separated from GFS via WAFSv7.0.0
This PR removes all traces of WAFS components, scripts, and workflow from this repository.

Refs #2591

# Type of change
- New feature (removes duplicate functionality)
- Maintenance (clean-up)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? YES
- Does this change require a documentation update? YES

# How has this been tested?
WAFS products will not be generated via GFS will be the test.

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: sorc/checkout.sh [hunk 1]
```diff
@@ -5,10 +5,7 @@ while getopts "o" option;
 do
  case $option in
   o)
-   echo "Received -o flag for optional checkout of GTG, will check out GTG with EMC_post"
-   checkout_gtg="YES"
-   checkout_wafs="YES"
-   gtg_git_args="--recursive"
+   echo "Received -o flag for operations"
```
Comment (aerorahul) [inline sorc/checkout.sh:8]: This is unnecessary now, but for the sake of continuity (and may be future operational specific options), I chose to keep it.

File: parm/config/config.base.nco.static [hunk 2]
```diff
@@ -49,7 +49,6 @@ export BASE_GIT="/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git"
 export DO_BUFRSND="YES"    # BUFR sounding products
 export DO_GEMPAK="YES"     # GEMPAK products
 export DO_AWIPS="YES"      # AWIPS products
-export WAFSF="YES"         # WAFS products
```
Comment (KateFriedman-NOAA) [inline parm/config/config.base.nco.static:52]: I thought this was staying as "NO". I'm fine with removing if not.

File: sorc/checkout.sh (hunk 1 above)
Comment (aerorahul) [reply sorc/checkout.sh:8]: Even if it was `YES`, it would do nothing. So.

File: parm/config/config.base.nco.static (hunk 2 above)
Comment (aerorahul) [reply parm/config/config.base.nco.static:52]: Even if it was `YES`, it would do nothing. So.

**Review by KateFriedman-NOAA — APPROVED** [review]
Left one comment to consider but otherwise this looks complete. Thanks @aerorahul !

**Comment by aerorahul** [comment]
@KateFriedman-NOAA Please update the release notes when you are able.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Comment by KateFriedman-NOAA** [comment]
@YaliMao-NOAA @HuiyaChuang-NOAA awaiting your inputs before we merge this into the release branch, thanks!

**Pull request merged**